doc/source/limitation.rst

```diff
@@ -6,30 +6,32 @@ Percona XtraDB Cluster Limitations
 
 The following limitations apply to |PXC|:
 
-* Replication works only with |InnoDB| storage engine.
-  Any writes to tables of other types, including system (``mysql.*``) tables,
-  are not replicated.
-  However, ``DDL`` statements are replicated in statement level,
-  and changes to ``mysql.*`` tables will get replicated that way.
-  So you can safely issue ``CREATE USER...``,
-  but issuing ``INSERT INTO mysql.user...`` will not be replicated.
-  You can enable experimental |MyISAM| replication support
-  using the :variable:`wsrep_replicate_myisam` variable.
+Replication works only with |InnoDB| storage engine.
+   Any writes to tables of other types, including system (``mysql.*``)
+   tables, are not replicated.
 
-* Unsupported queries:
+   However, ``DDL`` statements are replicated in statement level, and
+   changes to ``mysql.*`` tables will get replicated that way.  So you
+   can safely issue ``CREATE USER...``, but issuing ``INSERT INTO
```

Comment:
With 8.0, this is no longer true.  System tables are all InnoDB.  Although I haven't tested if the changing the tables directly replicates.